src/app/item-card/item-card.component.ts

```diff
@@ -0,0 +1,56 @@
+import { Component, Input, Output, EventEmitter } from "@angular/core";
+import { Item } from "../model/item.interface";
+
+@Component({
+  selector: "app-item-card",
+  templateUrl: "./item-card.component.html",
+  styleUrls: ["./item-card.component.scss"],
+})
+export class ItemCardComponent {
+  private maxDescriptionLength: number = 150;
```

Comment:
The ellipsis (...) on UI is not usually handled like this.
Take a look on the internet about it and let's talk!